kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/GroupMetadata.java

```diff
@@ -166,11 +167,11 @@ public static class GroupSummary {
      */
     @Data
     static class CommitRecordMetadataAndOffset {
-        private final Optional<Long> appendedBatchOffset;
+        private final Optional<PositionImpl> appendedBatchOffset;
```

Comment:
It's better to rename it with `appendedBatchPosition` since it doesn't represent an offset anymore.

Reply:
renamed